mscore/preferences.cpp

```diff
@@ -648,7 +648,7 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
       for (int idx = 0; idx < n; ++idx)
             exportAudioSampleRate->addItem(QString("%1").arg(exportAudioSampleRates[idx]));
 
-      restartWarningLanguage->setText("");
+      restartWarningLanguage->hide();
```

Comment:
You tried this? It could be that the visual effect is different since a hidden control will not be taken into account in the layout.